ios/UARCTModule/UrbanAirshipReactModule.m

```diff
@@ -528,4 +530,38 @@ - (RCTBridge *)bridge {
         reject(UARCTStatusUnavailable, @"Only available on iOS 10+", error);
     }
 }
+
+#pragma mark -
+#pragma mark Helper methods
+
+- (void)displayOverlayMessage:(UAInboxMessage *)message {
+    if (!self.factoryBlockAssigned) {
```

Comment:
Just to verify I understand this, the code will set the factory block regardless if any other code has set one. Not sure what other code would do that...

Reply:
Right. It's an unlikely scenario, since the app developer is going to be writing some flavor of JS instead of native code. It's technically possible that some random third party plugin could be messing around with Airship APIs but I think that's well within the boundary of unsupported integration.